examples/README.md

```diff
@@ -0,0 +1,16 @@
+# Examples
+
+This folder contains a couple of example files to use for testing the MLFMU tools and some FMUs that have been generated using this tool.
+Each folder is set up to contain:
+
+* `config`: containing the `.onnx` and `interface.json` files
+* `generated_fmu`: binary FMU files, to serve as examples
+* `<FmuName>`: source files for the FMU
```

Comment:
Currently we do not have a folder with the generated source for the `wind_to_power_pyspark` model. Should we add that?

Reply:
While we are chaning the examples folders I saw that the generated source for the other two example fmus are not updated with the newest verison. We could just update these while we are changing the generated sources in the examples folder? We could also fix this in another PR

Reply:
Yes, it is a bit different, because the initial source code for the wind FMUs (vs pyspark example) is in our internal repo with models, and then it has the FMU content here. I thus also wondered if we should include that source code as well (or not), given that the pyspark notebook is included.

Reply:
Add for pyspark case.